backend/src/main/java/pl/edu/agh/apdvbackend/mappers/group/group_members/MemberGroupNamesMapper.java

```diff
@@ -0,0 +1,19 @@
+package pl.edu.agh.apdvbackend.mappers.group.group_members;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.Set;
+import org.mapstruct.Mapper;
+import pl.edu.agh.apdvbackend.models.database.Group;
+
+@Mapper(componentModel = "spring")
+public abstract class MemberGroupNamesMapper {
+    public List<String> toOtherGroupNames(Set<Group> userGroups, Long groupId) {
+        return userGroups
+                .stream()
+                .filter(group -> !Objects.equals(group.getId(), groupId))
+                .map(Group::getName)
+                .sorted()
```

Comment:
potrzebujemy sortowania tutaj?

Reply:
`userGroups` jest `Set'em` i nazwy tam nie będą po kolei. Można to usunąć, ale wtedy `frontend` będzie odpowiedzialny za sortowanie tych nazw

Reply:
luźne pytanie XD

Reply:
Bardzo dobrze, trzeba uważnie czytać czyiś kod